### PR TITLE
fix issue with not seeing your own attachments

### PR DESCRIPTION
**Pull request by ZappoMan**
This fixes the bug where you can't see your own attachments.

This reverts some code that was part of this commit: https://github.com/highfidelity/hifi/pull/10255/commits/4511156d7bbba1cd558223eb14a10b6a2e21470f

@AndrewMeadows - was there a reason you made this change?

**Test Plan**
1. login
2. Attach something to yourself
3. Verify that you can see the attachment.
4. Verify that others can see the attachment.

### CODE REVIEW

File: interface/src/avatar/Avatar.cpp [hunk 1]
```diff
@@ -525,13 +525,11 @@ static TextRenderer3D* textRenderer(TextRendererType type) {
...
     auto avatarPayload = new render::Payload<AvatarData>(self);
     auto avatarPayloadPointer = Avatar::PayloadPointer(avatarPayload);
-    if (_skeletonModel->addToScene(scene, transaction)) {
-        _renderItemID = scene->allocateID();
-        transaction.resetItem(_renderItemID, avatarPayloadPointer);
-
-        for (auto& attachmentModel : _attachmentModels) {
-            attachmentModel->addToScene(scene, transaction);
-        }
+    _renderItemID = scene->allocateID();
```
Comment (samcake) [inline interface/src/avatar/Avatar.cpp]: WE should simply do this allocation and the subsequent scene transaction IF the _renderItemID is not valid.
I don't think this should be called all the time.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e87f8dfe45fd76e8d16e30e3ea5ec0cf5430caec available at: https://gist.github.com/79be8948c91bedee12837c7991e4223f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10386-e87f8df.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10386-e87f8df.dmg

File: interface/src/avatar/Avatar.cpp (hunk 1 above)
Comment (AndrewMeadows) [inline interface/src/avatar/Avatar.cpp]: So you're saying it should be:
```
if (_renderItemID.isInvalidID()) {
     _renderItemID = scene->allocateID();  
}
```

Comment (samcake) [inline interface/src/avatar/Avatar.cpp]: not everything, just the block doing the allocation & reset of the metaItem

**Comment by claudioshg** [comment]
testing

**Comment by claudioshg** [comment]
testing

File: interface/src/avatar/Avatar.cpp (hunk 1 above)
Comment (ZappoMan) [inline interface/src/avatar/Avatar.cpp]: cool - easy fix... this is the old original code... but I can improve it. what's the good test for invalid? is 0 invalid?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 43a177cf9a3c3d9e271d1d7c7cac4ee3023384d3 available at: https://gist.github.com/79be8948c91bedee12837c7991e4223f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10386-43a177c.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10386-43a177c.dmg

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10386-e87f8df.exe

PASS:
- Logged in
- Attached gloves from dev-welcome
- Visual confirmation of the gloves equipped by me, and one other person in the domain.
apr10386

**Pull request merged**